src/api/client/server.rs

```diff
@@ -95,12 +106,13 @@ mod test {
     use futures::Stream;
     use hashbrown::HashMap;
 
-    use crate::api::{
-        client::{session, Room},
-        control::Member,
+    use crate::{
+        api::{client::Room, control::Member},
+        conf::Conf,
     };
 
     use super::*;
+    use crate::conf::server::Server;
```

Comment:
This is not very ergonomic due to know how `conf` module is layouted. It's better to re-export stuff in `conf`, so I can just do `conff::Server`.